### ISSUE TITLE
Add nicer loading-state to v10-13

### ISSUE THREAD

**Issue by kasparboelkjeldsen** [kasparboelkjeldsen/Knowit.Umbraco.InstantBlockPreview#20]
Currently as the frontend is waiting on the first api response in Instant Block Preview for version 13 and lower, it will just output the text "loading..." 

It would be nice to have a proper loading indicator/spinner

hacktober-fest quickstart
Clone down the project, run the /Umbraco/Umbraco.-14.2 dotnet project

In Knowit.Umbraco.Bellissima.InstantBlockPreview.Frontend run "npm i" and "npm watch" and choose the Umbraco-14.2 in the options.
